models/pipeline.js

```diff
@@ -101,15 +104,19 @@ module.exports = {
      * @property create
      * @type {Joi}
      */
-    create: Joi.object(CHECKOUT_URL).label('Create Pipeline'),
+    create: Joi.object(mutate(CREATE_MODEL, [], [
+        'checkoutUrl', 'rootDir'
```

Comment:
for create, `checkoutUrl` should be required

Reply:
oops, fixed